public_html/styling/index.css

```diff
@@ -0,0 +1,388 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+    font-size: 16px;
+}
+
+
+/* Your code starts here! */
+* {
+    box-sizing: border-box;
+}
+
+.container {
+    margin: 0 auto;
+    max-width: 860px;
+}
+
+header {
+	align-items: center;
+	justify-content: space-between;
+	display: flex;
+}
+
+header a {
+	align-items: center;
+	display: flex;
+
+}
+
+.nav  {
+		width: 70%;
+    margin-top: 3%;
+    justify-content: space-between;
+    align-items: center;
+    display: flex;
+}
+
+.nav a {
+		text-decoration: none;
+		cursor: pointer;
+    color: grey;
+}
+
+
+
+/* Top Content Box */
+.content1 {
+		margin-top: 4%;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    border-bottom: 1px solid grey;
+}
+
+/* Top Header and Button */
+.contentbox1 {
+    text-align: center;
+    margin: 0 8% 0 0;
+    align-items: center;
+}
+
+.contentbox1 h2 {
+    font-size: 380%;
+}
+
+.contentbox1 button {
+		cursor: pointer;
+    border-radius: 12%;
+    width: 80%;
+		height: 40px;
+    border: 1px solid grey;
+}
+
+.contentbox1 button:hover {
+		background-color: #222222;
+		color: white;
+		transition: 0.5s;
+		transform: scaleX(1.08);
+    border-radius: 5px;
+    width: 80%;
+		height: 40px;
+    border: 1px solid grey;
+}
+
+/* Top image */
+.contentbox2 {
+    margin: 4% 0 4% 8%;
+}
+
+/* First Row of Article content */
+.content2 {
+    margin-top: 6%;
+    justify-content: space-around;
+    display: flex;
+}
+
+.textcard {
+    width: 40%;
+}
+
+/* Img in middle of Article content */
+.content3 {
+    justify-content: space-between;
+    display: flex;
+    align-items: center;
+    height: 50%;
```

Comment:
This should work without a height property

Reply:
Thanks I deleted it, you should see it revised in RWD 2